test/v3/addLiquidityBoosted/addLiquidityPartialBoosted.integration.test.ts

```diff
@@ -50,7 +50,8 @@ const parentBptToken = new Token(
 const usdtToken = new Token(chainId, USDT.address, USDT.decimals);
 const daiToken = new Token(chainId, DAI.address, DAI.decimals);
 
-describe('V3 add liquidity partial boosted', () => {
+// TODO: pending test pool to be created/initialized
+describe.skip('V3 add liquidity partial boosted', () => {
```

Comment:
I already pinged @mendesfabio - as soon as the test pool is created/initialized, we should be able to update addresses and hopefully no other changes will be required

Reply:
Test is now passing after updating with identical partially boosted pool that I deployed with [this script](https://github.com/balancer/b-sdk/blob/deploy-11/examples/createAndInitPool/createAndInitPoolV3.ts)